sweepbatcher/sweep_batch.go

```diff
@@ -2495,3 +2588,10 @@ func clampBatchFee(fee btcutil.Amount,
 
 	return fee
 }
+
+// dustLimitForPkScript returns the dust limit for a given pkScript.
+func dustLimitForPkScript(pkscript []byte) btcutil.Amount {
```

Comment:
I propose to add a small unit test for a couple of address types (e.g. p2wpkh and p2tr).